src/core/server/plugins/discovery/plugin_manifest_parser.ts

```diff
@@ -121,6 +121,15 @@ export async function parseManifest(
     );
   }
 
+  if (!manifest.owner || !manifest.owner.name || typeof manifest.owner.name !== 'string') {
+    throw PluginDiscoveryError.invalidManifest(
+      manifestPath,
+      new Error(
+        `Plugin manifest for "${manifest.id}" must contain an "owner" property, which includes a nested "name" property.`
+      )
+    );
+  }
```

Comment:
Would you be willing to also add a check for this specific condition to the existing invalid manifest test? https://github.com/stacey-gammon/kibana/blob/76ce9350928a06ffe960f4d2e439299ecf3b5be4/src/core/server/plugins/discovery/plugins_discovery.test.ts#L179

Reply:
Done!